easybuild/easyblocks/n/nwchem.py

```diff
@@ -98,12 +102,14 @@ def configure_step(self):
                 self.log.debug("Contents of %s: %s", os.path.dirname(self.local_nwchemrc),
                                os.listdir(os.path.dirname(self.local_nwchemrc)))
 
-                if os.path.islink(self.home_nwchemrc) and not os.path.samefile(self.home_nwchemrc, self.local_nwchemrc):
-                    raise EasyBuildError("Found %s, but it's not a symlink to %s. "
-                                         "Please (re)move %s while installing NWChem; it can be restored later",
-                                         self.home_nwchemrc, self.local_nwchemrc, self.home_nwchemrc)
+                if os.path.islink(self.home_nwchemrc):
+                    home_nwchemrc_target = os.readlink(self.home_nwchemrc)
+                    if home_nwchemrc_target != self.local_nwchemrc:
+                        raise EasyBuildError("Found %s, but it's not a symlink to %s. "
```

Comment:
Doesn't it make sense to move it and restore it later automatically?

Reply:
I'm not sure... What if multiple `NWChem` builds are being done in parallel on the system for example?

Also, if there's a symlink in place that's different from what EB expects, it was most likely created by the user for a particular version of `NWChem`.

It's safer to be explicit about this imho.

Reply:
Valid points.